src/metabase/middleware.clj

```diff
@@ -262,8 +262,10 @@
 (defn log-api-call
   "Middleware to log `:request` and/or `:response` by passing corresponding OPTIONS."
   [handler & options]
-  (fn [request]
-    (if-not (api-call? request)
+  (fn [{:keys [uri], :as request}]
+    (if (or (not (api-call? request))
```

Comment:
because calls to fetch `index.html`, `favicon.ico`, `app.js`, and various image assets are generally not that interesting